README.md

```diff
@@ -393,6 +393,16 @@ Different formatting of identifiers (e.g. class names, keyframes, CSS Vars, etc)
 
 Each integration will set a default value based on the configuration options passed to the bundler.
 
+### esbuildOptions (Vite)
+The Vite plugin uses esbuild internally to compile `.css.ts` files before running them to extract styles. You can pass additional options here to customize that process.
+
+Accepts an esbuild build options object: https://esbuild.github.io/api/#build-api which is deep merged with the default options.
+
+### internalEsbuildOptions (Vite)
```

Comment:
I can see why you made this one different but I feel like it might be easier to just name it `esbuildOptions` in all the plugins. Makes the docs much cleaner too.